Update shipyard setting docs (#860)
submariner-io/submariner-website: src/content/development/shipyard/settings/_index.en.md

    and **`ovn`**.
  * **`nodes`**: A space separated list of nodes to deploy, supported types are **`control-plane`** and **`worker`**.
  * **`submariner`**: If Submariner should be deployed, set to **`true`**. Otherwise, leave unset (or set to **`false`** explicitly).

>   * **`gateways`**: Number of gateway nodes that needs to be deployed globally or on per cluster level.

Reviewer: This is in the “Global and/or cluster specific” section, so “globally or on per cluster level” is specified by the section:

```suggestion
  * **`gateways`**: Number of gateway nodes to deploy.
```

Author: Done

    submariner: false
    nodes: control-plane
```

The following settings file deploys two clusters.

> The second cluster will deploy one control node and three worker nodes and 2 nodes will be labeled as gateway nodes.

Reviewer: ```suggestion
The second cluster will deploy one control node and three worker nodes and two nodes will be labeled as gateway nodes.
```

Author: Done

> As no **`gateways`** specified on global and cluster1 level, cluster1 will get a default number of 1 gateway node.

Reviewer: ```suggestion
As no **`gateways`** setting is specified either globally or for the first cluster specifically, the first cluster will get the default single gateway node.
```

Author: Done

    submariner: false
    nodes: control-plane
```

The following settings file deploys two clusters.

> The second cluster will deploy one control node and three worker nodes and two nodes will be labeled as gateway nodes.

Reviewer 2: ```suggestion
The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
```

As no **`gateways`** setting is specified either globally or for the first cluster specifically,

> the first cluster will get the default single gateway node.

Reviewer 2: ```suggestion
the first cluster will get have a single gateway node by default.
```

> The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
> As no **`gateways`** setting is specified either globally or for the first cluster specifically,
> the first cluster will get have a single gateway node by default.

Reviewer 2: ```suggestion
As no **`gateways`** setting is specified either globally or for the first cluster specifically,
the first cluster will get have a single gateway node by default.
The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
```

Somehow reading "the second cluster" as the first paragraph, and then "... the first cluster" as the second, feels wrong to me.
Ordering them makes it better IMO
